pkg/debug/spec.sh

```diff
@@ -215,6 +215,17 @@ get_modem_atport() {
     return 1
 }
 
+add_description() {
+    DEVICE_TYPE="$1"
+    if [ -n "$verbose" ]; then
+        desc=$(lspci -D | grep "${DEVICE_TYPE}")
```

Comment:
Since the argument is the PCI bus/slot (but didn't check all callers) wouldn't it make sense to use -s "$1" instead of grep?